system_tests/bigquery.py

```diff
@@ -185,6 +187,7 @@ def test_patch_table(self):
         self.assertEqual(table.friendly_name, 'Friendly')
         self.assertEqual(table.description, 'Description')
 
+    @retry(Exception, tries=3, delay=20)
```

Comment:
I'm not 100% sure here but I think for many of these we want to retry a specific line in the method rather than the entire test case. Things like set-up probably aren't worth repeating / repeating them will increase the probability of failure.